js/views/modals/listingDetail/Listing.js

```diff
@@ -168,13 +282,30 @@ export default class extends BaseModal {
         shipsFreeToMe: this.shipsFreeToMe,
         ownListing: this.model.isOwnListing,
         displayCurrency: app.settings.get('localCurrency'),
+        shipsFromCountry: app.settings.get('country'),
       }));
 
       super.render();
 
       this._$deleteListing = null;
       this._$shipsFreeBanner = null;
       this._$popInMessages = null;
+      this._$photoSection = null;
```

Comment:
There's a bunch more you need to reset to null here, basically anything that you're getting via this type of getter:

```
  get $shippingOptions() {
    return this._$shippingOptions ||
      (this._$shippingOptions = this.$('.js-shippingOptions'));
  }
```

Otherwise, upon re-render, those will point to elements that are no longer in the DOM.

That is why currently if you Edit the listing's shipping options and click 'refresh' in the pop-up message, the shipping options are ends up being empty:

![image](https://cloud.githubusercontent.com/assets/794517/21122672/8a9c3892-c088-11e6-9e0d-4988beeb780d.png)